go.mod

```diff
@@ -65,6 +68,7 @@ require (
 )
 
 require (
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
there's still an indirect dep? is that from sdk and it'll go away soon?

Reply:
The SDK supports Gogo for unmarshaling user-supplied protos. I have no idea if we need to continue supporting it; @cretz do we still need to [support gogo in our payload converters](https://github.com/tdeebswihart/temporal-sdk-go/blob/master/converter/proto_payload_converter.go#L32) or can it be removed entirely?

Reply:
I think we can remove it. It sort of breaking change but I doubt that anyone use gogo to generate in/out params.

Reply:
The SDK team has mentioned we need to keep this as some of our customers use Gogo/protobuf